submitter/tx_submitter.go

```diff
@@ -130,14 +165,16 @@ func (s *TxSubmitter) preCheck(event *model.Event) error {
 }
 
 func (s *TxSubmitter) checkSubmitStatus(challengeId uint64) error {
-	attestedChallengeId, err := s.executor.QueryLatestAttestedChallengeId()
-	logging.Logger.Infof("latest attested challengeId: %d", attestedChallengeId)
+	attestedChallengeIds, err := s.executor.QueryLatestAttestedChallengeIds()
```

Comment:
Call `QueryLatestAttestedChallengeIds` is also fine.
However, you may need to review `QueryLatestAttestedChallengeIds`.